packages/node/src/indexer/store.service.test.ts

```diff
@@ -64,7 +64,7 @@ describe('Store service integration test', () => {
 
     tempDir = (projectService as any).project.root;
 
-    const [result] = await sequelize.query(
+    const result = await sequelize.query<{ table_name: string }>(
```

Comment:
Is this right? I thought it would be 
```suggestion
    const [result] = await sequelize.query<[{ table_name: string }]>(
```